dev/com.ibm.ws.jdbc_fat_postgresql/fat/src/com/ibm/ws/jdbc/fat/postgresql/FATSuite.java

```diff
@@ -10,16 +10,91 @@
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.utils.ExternalTestService;
 
 @RunWith(Suite.class)
 @SuiteClasses({
-                AlwaysPassesTest.class, // needed for OSes that do not have docker installed
                 PostgreSQLTest.class,
 })
 public class FATSuite {
+
+    @BeforeClass
+    public static void verifyDockerAvailable() throws Exception {
+        // TODO: Once this mechanism has matured a bit, move it to fattest.simplicity for more general use
+
+        // Use local Docker install if running locally.
+        // Devs can override this by specifying -Dfat.test.DOCKER_HOST=tcp://example.com:2345 when they launch the FAT
+        if (FATRunner.FAT_TEST_LOCALRUN) {
+            if (!System.getProperty("fat.test.DOCKER_HOST", "").isEmpty())
+                System.setProperty("DOCKER_HOST", System.getProperty("fat.test.DOCKER_HOST"));
+            boolean dockerHostSet = !System.getProperty("DOCKER_HOST", "").isEmpty() ||
+                                    (System.getenv("DOCKER_HOST") != null && !System.getenv("DOCKER_HOST").isEmpty());
+            File testcontainersConfigFile = new File(System.getProperty("user.home"), ".testcontainers.properties");
+            if (!testcontainersConfigFile.exists())
+                return;
+            Properties testcontainerProps = new Properties();
+            try (FileInputStream fis = new FileInputStream(testcontainersConfigFile)) {
+                testcontainerProps.load(fis);
+            }
+            String currentStrategy = testcontainerProps.getProperty("testcontainers.properties");
+            if (!dockerHostSet && "org.testcontainers.dockerclient.EnvironmentAndSystemPropertyClientProviderStrategy".equals(currentStrategy)) {
+                Log.info(c, "verifyDockerAvailable", "No DOCKER_HOST set in env or sysprops, but env/sysprop Docker client strategy found in ~/.testcontainers.properties.");
+                Log.info(c, "verifyDockerAvailable", "Attempting to reset Testcontainers properties file by deleting " + testcontainersConfigFile.getAbsolutePath());
+                Files.delete(testcontainersConfigFile.toPath());
+            } else if (dockerHostSet && !"org.testcontainers.dockerclient.EnvironmentAndSystemPropertyClientProviderStrategy".equals(currentStrategy)) {
+                Log.info(c, "verifyDockerAvailable", "DOCKER_HOST set in env/sysprops, but env/sysprop strategy not set in ~/.testcontainers.properties.");
+                Log.info(c, "verifyDockerAvailable", "Attempting to reset Testcontainers properties file by deleting " + testcontainersConfigFile.getAbsolutePath());
+                Files.delete(testcontainersConfigFile.toPath());
+            }
+            return;
+        } else {
+            // Running in a remote build, check Consul for an external Docker service
+            for (ExternalTestService dockerService : ExternalTestService.getServices(10, "aguibert-test-docker-host")) {
```

Comment:
This is a placeholder service name until @websterc87 is able to help me set up a proper service in Consul